src/group_operations/addition.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    AdditionGroupOperation <: AbstractGroupOperation
+
+A group operation that is realised introducing defaults that fall back
+to `+` and `-` being overloaded, for example
+`_compose(G::LieGroup{𝔽,AdditionGroupOperation}, a,b) = a+b`
+"""
+struct AdditionGroupOperation <: AbstractGroupOperation end
+
+#
+#
+# Handle interactions of `+` and `-` with the identity element
+# still necessary? This is handled in compose now anyways
+Base.:+(e::Identity{AdditionGroupOperation}) = e
+Base.:+(e::Identity{AdditionGroupOperation}, ::Identity{AdditionGroupOperation}) = e
+Base.:+(::Identity{AdditionGroupOperation}, g) = g
+Base.:+(g, ::Identity{AdditionGroupOperation}) = g
+
+Base.:-(e::Identity{AdditionGroupOperation}) = e
+Base.:-(e::Identity{AdditionGroupOperation}, ::Identity{AdditionGroupOperation}) = e
+Base.:-(::Identity{AdditionGroupOperation}, g) = -g
+Base.:-(::Any, ::Identity{AdditionGroupOperation}) = g
+
+Base.:*(e::Identity{AdditionGroupOperation}, p) = e
+Base.:*(::Any, e::Identity{AdditionGroupOperation}) = e
+Base.:*(e::Identity{AdditionGroupOperation}, ::Identity{AdditionGroupOperation}) = e
+
+function _compose!(G::LieGroup{𝔽,AdditionGroupOperation}, k, g, h) where {𝔽}
+    ManifoldsBase.copyto!(G, k, g + h)
+    return k
+end
+
+"""
+"""
+Base.exp(
+    ::LieGroup{𝔽,AdditionGroupOperation}, ::Identity{AdditionGroupOperation}, X
+) where {𝔽}
+
+function ManifoldsBase.exp!(
+    G::LieGroup{𝔽,AdditionGroupOperation},
+    g,
+    ::Identity{AdditionGroupOperation},
+    X,
+    t::Number=1,
+) where {𝔽}
+    return copyto!(G, g, X)
+end
+
+function identity_element!(::LieGroup{𝔽,AdditionGroupOperation}, e) where {𝔽}
+    return fill!(e, 0)
+end
+
+function inv!(G::LieGroup{𝔽,AdditionGroupOperation}, h, g) where {𝔽}
+    return copyto!(G, h, -g)
+end
+
+function is_identity(G::LieGroup{𝔽,AdditionGroupOperation}, h; kwargs...) where {𝔽}
+    return ManifoldsBase.isapprox(G, Identity{AdditionGroupOperation}(), h; kwargs...)
+end
+function is_identity(
+    G::LieGroup{𝔽,AdditionGroupOperation}, h::Identity{AdditionGroupOperation}; kwargs...
+) where {𝔽}
+    return true
+end
+function is_identity(
+    G::LieGroup{𝔽,AdditionGroupOperation}, h::Identity; kwargs...
+) where {𝔽}
+    return true
```

Comment:
Shouldn't this be `false`? If `h` is addition identity, the method above should catch it and for non-addition identities I'd expect `false` here.

Reply:
Yes, correct. I hit one of these logical things just now in exp/log fallbacks, and will work through the interface functions while writing the test functions.
This could probably also already be checked on the interface level and no longer here (same Op-> true, different -> false)

Reply:
Yes, this check can be interface-level for sure.

Reply:
Fixed it.